tests/browser/batch_gift_entry/create_new_account_and_payment.robot

```diff
@@ -0,0 +1,63 @@
+*** Settings ***
+
+Resource        tests/NPSP.robot
+Library           DateTime
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Create a new account and enter payment information
+    #Create a new account and enter payment information, then process batch
+    [tags]  unstable
+    ${ns} =  Get NPSP Namespace Prefix
+    &{batch} =       API Create DataImportBatch    ${ns}Batch_Process_Size__c=50    ${ns}Batch_Description__c=Created via API    ${ns}Donation_Matching_Behavior__c=Single Match or Create    ${ns}Donation_Matching_Rule__c=donation_amount__c;donation_date__c    ${ns}RequireTotalMatch__c=false    ${ns}Run_Opportunity_Rollups_while_Processing__c=true   ${ns}GiftBatch__c=true    ${ns}Active_Fields__c=[{"label":"Donation Amount","name":"${ns}Donation_Amount__c","sObjectName":"Opportunity","defaultValue":null,"required":true,"hide":false,"sortOrder":0,"type":"number","options":null},{"label":"Donation Date","name":"${ns}Donation_Date__c","sObjectName":"Opportunity","defaultValue":null,"required":false,"hide":false,"sortOrder":1,"type":"date","options":null}]     
+    Select App Launcher Tab   Batch Gift Entry
+    # Click Link  &{batch}[Name]
+    Click Link With Text    &{batch}[Name]
+    Wait For Locator    bge.title    Batch Gift Entry
+    Select Value From BGE DD    Donor Type    Account
+    ${acc_name} =  Generate Random String
+    Populate Address    Search Accounts    ${acc_name}
+    Click Element    title:Search Accounts...
+    Wait For Locator    record.edit_button    New Account
+    Click Element    title=New Account
+    Select Record Type         Organization
+    Populate Form
+    ...                        Account Name=${acc_name}
+    Click Modal Button         Save    
+    Wait Until Modal Is Closed
+    Fill BGE Form
+    ...                       Donation Amount=20
+    Select BGE Date Picker    Donation Date
+    Click BGE Button    Today
+    Click BGE Button       Save
+    Wait For Locator    bge.title    Batch Gift Entry
+    Reload Page
+    Verify Row Count    1 
+    Wait For Locator    bge.edit_button    Donation Amount
+    SeleniumLibrary.Element Text Should Be    //td[@data-label="Donation"]//lightning-formatted-url    ${Empty}
```

Comment:
I got an error when using this keyword because it was in multiple libraries and said use instead like this. I think it was in selenium and built in. I dont remember exactly.